### DISCUSSION TITLE
Toggle option for old style grey checkmarks / new style hollow colored checkmarks

### DISCUSSION THREAD

**Discussion by HoobieGits** [iSoron/uhabits#1149] [Feature Requests]
As of the latest update, I do not like the change in the appearance of the hollow checkmarks. 

By "hollow checkmarks" I mean the checkmarks that appear when a habit is less frequent than daily, e.g. 3x/week, and I have checked it for the past interval (e.g. it's Thursday and I checked it on Monday, Tuesday, and Wednesday). 

The new hollow checkmarks have the same color as the manual checkmarks, with a white space in their middle. They take up too much visual space and my eyes are drawn to them instead of to the grey Xs and colored checkmarks. They make the app harder to use because it takes more attention for me to figure out whether I have marked a box as a check or not.

I want the old style back for hollow checkmarks - grey checkmarks that look similar to the grey Xs. Would you please make an option to toggle this appearance in the app's settings?

Thank you.

**Comment by hiqua** [comment]
I don't think adding options is a good option if we can avoid it.

But we could change the hollow checkmark to make it less prominent, I can think of:
* making it smaller
* dimming its color
* changing the shape (circle?)

**Reply by HoobieGits** [reply]
I'd like the hollow/grey checkmarks to be easier to ignore. Between grey Xs, colored checks, skips, and ?s, they are functionally the least important marker. 

Yes, the hollow checkmark is larger than the colored checkmark, so making the hollow checkmark smaller would go some distance towards solving the problem.

Changing the color, or especially the shape, would introduce yet more information into the display. Please, please don't do that. 

I'd just prefer the grey checkmark back, because it's the least visually distinct. I don't understand why that option was taken away; it fit perfectly into how I was using the app. Why was it removed? 

I furthermore don't understand why adding it as an option isn't a good idea. Would you explain that to me? 

Thank you.

**Reply by hiqua** [reply]
> Yes, the hollow checkmark is larger than the colored checkmark, so making the hollow checkmark smaller would go some distance towards solving the problem.

I hadn't noticed, and you're right, the hollow checkmark is the outer line of the checkmark. It'd be better to use the inner one, if that makes sense.

> I don't understand why that option was taken away; it fit perfectly into how I was using the app. Why was it removed?

Was there ever an option?

> I furthermore don't understand why adding it as an option isn't a good idea. Would you explain that to me?

It's just a general remark, more options means more things to test, and more bugs. Of course there are cases where they're really needed, but if we can find a compromise that would be better.

**Reply by HoobieGits** [reply]
I agree, using the inner line of the checkmark would go some distance towards solving the problem. Thanks.

> Was there ever an option?

I misspoke when I used the word "option." To be more precise, I meant something like "the appearance that was working for me." 

I hear that you'd prefer not to add options unless you need to, because that adds things to test and bugs. At the same time, I'm frustrated because the app was working perfectly for me before, and now it's less convenient and I don't have any choice to revert its appearance. Frankly, if I could I would roll back the last update. 

Thanks for your help.
